Core/Graphics/Source/Graphics.cpp

```diff
@@ -262,19 +262,34 @@ namespace Babylon
 
     Graphics::~Graphics() = default;
 
+    template<>
+    void Graphics::UpdateWindow<void*>(void* windowPtr)
+    {
+        m_impl->SetNativeWindow(windowPtr, nullptr);
+    }
+
+    template<>
+    void Graphics::UpdateWindow<void*, void*>(void* windowPtr, void* windowTypePtr)
+    {
+        m_impl->SetNativeWindow(windowPtr, windowTypePtr);
+    }
+
```

Comment:
Is the following still necessary? If so, does it need to change?

https://github.com/BabylonJS/BabylonNative/blob/03a5c2dafe96c0fe354fc3a462ffa86184762e4c/Core/Graphics/Source/Graphics.cpp#L26-L30

Reply:
good catch, didn't see these before. We no longer need them